pkgs/development/web/cypress/cypress-example-kitchensink/cypress-example-kitchensink.nix

```diff
@@ -0,0 +1,17 @@
+# This file has been generated by node2nix 1.9.0. Do not edit!
+
+{pkgs ? import <nixpkgs> {
```

Comment:
This should use the relative path.

Reply:
This default value is generated dead code. The caller never uses the default.
I'd like to avoid patching generated code when possible.